include/swift/AST/DiagnosticsSema.def

```diff
@@ -5431,9 +5431,12 @@ ERROR(isolated_default_argument_context,none,
 ERROR(conflicting_default_argument_isolation,none,
       "default argument cannot be both %0 and %1",
       (ActorIsolation, ActorIsolation))
-ERROR(distributed_actor_needs_explicit_distributed_import,none,
-      "'Distributed' module not imported, required for 'distributed actor'",
-      ())
+ERROR(distributed_decl_needs_explicit_distributed_import,none,
+      "%0 declaration requires importing Distributed module explicitly",
```

Comment:
Maybe “%kind0 requires explicit import of Distributed module”?

Also small thing - you can now pass `ValueDecl *` instead of `DescriptiveDeclKind`

Reply:
That sounds nice, thank you :) Will do the ValueDecl* as well, thx!